pick.lic

```diff
@@ -112,8 +112,9 @@ class LockPicker
   end
 
   def buy_lockpick(type)
-    fput("buy #{type} lockpick")
-    fput("offer #{@lockpick_costs[type]}")
```

Comment:
They're still used in refill_ring() on line 98 where we pull money out of the bank to pay for X lockpicks. I think it might be useful to have them there as an upper bound on the cost, since we don't know what the discount will be